api/edge_api/identities/serializers.py

```diff
@@ -189,7 +186,7 @@ def save(self, **kwargs):
                 "environment_api_key": identity.environment_api_key,
                 "identity_id": identity.id,
                 "identity_identifier": identity.identifier,
-                "changed_by_user_id": request.user.id,
+                "changed_by": str(request.user),
```

Comment:
@gagantrivedi changing the signature here will be a problem during the deployment because the old task processor instances will not be able to execute the function with the updated arguments. We need to resolve this please in a new PR. 

I suggest that we just add a new argument `changed_by` to the task function and update the calling code to only send `changed_by_user_id` if `request.user` is an `FFAdminUser` and then ignore that field in the actual task handler. 

Note: we should probably consider adding some kind of versioning logic to the task processor to handle this in the future too.